dali/base/dadfs.hpp

```diff
@@ -646,6 +646,7 @@ interface IDistributedFileDirectory: extends IInterface
     virtual IUserDescriptor* queryDefaultUser()=0;
     virtual SecAccessFlags getNodePermissions(const IpAddress &ip,IUserDescriptor *user,unsigned auditflags=0)=0;
     virtual SecAccessFlags getFDescPermissions(IFileDescriptor *,IUserDescriptor *user,unsigned auditflags=0)=0;
+    virtual SecAccessFlags getDropZoneScopePermissions(const char *dropZoneNameOrIP,const char *dropZonePath,bool isPlane,IUserDescriptor *user,unsigned auditflags=0)=0;
```

Comment:
Inconsistency in naming of isPlane in interface, isExternalPlane in implementation

Reply:
It may be more accurate using the isPlane.

Reply:
I agree